OpenSim/Simulation/Model/CMCActuatorSubsystem.cpp

```diff
@@ -148,11 +148,16 @@ void CMCActuatorSubsystemRep::setSpeedTrajectories(FunctionSet *aSet) {
     // controls
     const CoordinateSet& coords = _model->getCoordinateSet();
     for (int i = 0; i < nq; ++i) {
-        // the last argument to setValue is a bool to enforce kinematic constraints
-        // or not. It is being set to true when we set the last coordinate value.
-        coords[i].setValue(mutableCompState, _qWork[i] + _qCorrections[i], i==(nq-1));
+        // the last argument to setValue, a bool to enforce constraints,
+        // is false since values come from a _qSet of splined desired
+        // kinematics formed from formCompleteStorages, which enforces
+        // model constraints.
+        coords[i].setValue(mutableCompState, _qWork[i] + _qCorrections[i], false);
         coords[i].setSpeedValue(mutableCompState, _uWork[i] + _uCorrections[i]);
     }
+    // projectQ is sufficient to satisfy constraints perturbed by _qCorrections
+    _model->getMultibodySystem().projectQ(mutableCompState,
+        getModel()->get_assembly_accuracy());
```

Comment:
I suggest either updating the documentation for the `assembly_accuracy` property or renaming the property to `constraint_accuracy`.

Reply:
The `assembly_accuracy` property name and description is more or less correct. I am using the `assembly_accuracy` as the accuracy for satisfying constraint equations using `projectQ()` instead of setting it arbitrarily. If we consider the constraint solution should be more stringent to achieve the desired coordinate accuracy, then perhaps I should update this to be:  `getModel()->get_assembly_accuracy()/10` since this is how the `SimTK::Assembler` sets the error tolerance on constraints from the user provided accuracy.

Reply:
> The assembly_accuracy property name and description is more or less correct. I am using the assembly_accuracy as the accuracy for satisfying constraint equations using projectQ() instead of setting it arbitrarily.

Sounds good. Developers might not realize this accuracy property is used outside of the AssemblySolver, so perhaps a non-doxygen comment could be helpful. But this is a minor point. I'll approve the PR.

If, in the future, we find we need more fine-grained control, perhaps we can add a property to CMCTool for the `projectQ()` accuarcy.